packages/blocks/src/blocks/CoreImage.tsx

```diff
@@ -0,0 +1,88 @@
+import { gql } from '@apollo/client';
+import React from 'react';
+import { Image } from 'next/image';
+import { useBlocksTheme } from '../components/WordPressBlocksProvider.js';
+import { ContentBlock } from '../components/WordPressBlocksViewer.js';
+import { getStyles } from '../utils/index.js';
+
+export type CoreImageFragmentProps = ContentBlock & {
+  attributes: {
+    align?: string;
+    alt?: string;
+    anchor?: string;
+    borderColor?: string;
+    caption?: string;
+    className?: string;
+    width?: string;
+    url?: string;
+    title?: string;
+    style?: string;
+    src?: string;
+    sizeSlug?: string;
+    rel?: string;
+    lock?: string;
+    linkTarget?: string;
+    linkDestination?: string;
+    linkClass?: string;
+    id?: string;
+    href?: string;
+    height?: string;
+    cssClassName?: string;
+  };
+};
+
+export function CoreImage(props: CoreImageFragmentProps) {
+  const theme = useBlocksTheme();
+  const style = getStyles(theme, { ...props });
+  const { attributes } = props;
+
+  return (
+    <figure>
+      <Image
+        style={style}
+        src={attributes.src}
+        width={attributes.width}
+        height={attributes.height}
+        alt={attributes.alt}
+        className={attributes?.cssClassName}
+      />
+    </figure>
```

Comment:
We'll want to conditionally include the caption as well. Something like:

```jsx
{attributes?.caption && (
  <figcaption>{attributes.caption}</figcaption>
)}
```

Reply:
I'm afraid this has angered my unit testing, @blakewilson! I'll check in with you Thursday. I appreciate it